.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,108 @@
+name: Fuzz Testing
+
+on:
+  push:
+    branches:
+      - main
+      - release-*
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+      - '**/OWNERS'
+      - '**/MAINTAINERS'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+      - '**/OWNERS'
+      - '**/MAINTAINERS'
+
+env:
+  FUZZ_TIME: "120s"  # Run each fuzzer for 2 minutes
+
+jobs:
+  fuzzing:
+    name: Go Fuzzing Tests
+    runs-on: ubuntu-22.04
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+
+      - name: Set up Go
+        uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5
+        with:
+          go-version: 1.22.x
```

Comment:
Minor version 22 of Go is pretty outdated. We should be using 24 or 25 now.

The ".x" looks odd. It is not documented as acceptable in https://github.com/actions/setup-go#supported-version-syntax . In common parlance, ".x" is used to indicate a uncertainty. I think that we want certainty.

The documentation for this action mentions some other possibilities too. I think that the most appropriate would be to use the latest stable version of Go.